server/apps/microservices/src/processors/machine-learning.processor.ts

```diff
@@ -13,30 +13,30 @@ const immich_machine_learning_url = process.env.IMMICH_MACHINE_LEARNING_URL || '
 
 @Processor(QueueNameEnum.MACHINE_LEARNING)
 export class MachineLearningProcessor {
+  readonly logger = new Logger(MachineLearningProcessor.name);
+
   constructor(
-    @InjectRepository(SmartInfoEntity)
-    private smartInfoRepository: Repository<SmartInfoEntity>,
+    @InjectRepository(AssetEntity)
+    private assetRepository: Repository<AssetEntity>,
+
+    @InjectRepository(TagEntity)
+    private tagRepository: Repository<TagEntity>,
```

Comment:
I see you opted to not use the TagRepository you created in the other PR.

Reply:
I cannot since those are in the core immich server, so we cannot share code here.

Reply:
Maybe we can explore, in a future PR, moving the repository classes to the database module.